info: CustomCategory_TestSignalWithCustomSerializedPayloadType[441733974]
      Handling signal with payload TestSignalWithCustomSerializedPayloadType { Payload = TestSignalWithCustomSerializedPayloadTypePayload { Payload = 10 } } (Signal ID: test-message-id, Trace ID: test-trace-id)
fail: CustomCategory_TestSignalWithCustomSerializedPayloadType[712209902]
      An exception occurred while handling signal after 123.4560ms (Signal ID: test-message-id, Trace ID: test-trace-id)
      Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals+TestException: test exception
         at Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomSerializedPayloadTypeHandler.Handle(TestSignalWithCustomSerializedPayloadType signal, CancellationToken cancellationToken) in {ProjectDirectory}Signalling/LoggingMiddlewareTestSignals.cs:line 659
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Signalling/LoggingSignalMiddleware.cs:line 51
         at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx)
         at Conqueror.Signalling.SignalPipelineRunner`1.<>c__DisplayClass3_2.<Execute>b__1(TSignal signal, CancellationToken token) in {SolutionDirectory}src/Conqueror/Signalling/SignalPipelineRunner.cs:line 36

